share/src/main/java/io/zensoft/share/config/Facebook/facebookConfig

```diff
@@ -0,0 +1,6 @@
+appId=1482536371851716
```

Comment:
May be we should keep this file in resources folder? how do you think @derkenbaev